test/extended/util/db/mysql.go

```diff
@@ -40,7 +40,7 @@ func (m MySQL) IsReady(oc *util.CLI) (bool, error) {
 		"mysqladmin -h 127.0.0.1 -uroot ping").Output()
 	if err != nil {
 		switch err.(type) {
-		case *exec.ExitError:
+		case *ExitError:
```

Comment:
@mfojtik you broke the crap out of our extended tests when you changed the return type of .OutPut() in https://github.com/openshift/origin/pull/10130
:(